engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -2198,7 +2204,176 @@ class RuntimeVisualizationsTest
                 "Enso_Test.Test.Visualisation",
                 "Enso_Test.Test.Visualisation",
                 "incAndEncode"
+              ),
+              Vector("2", "3")
```

Comment:
What other types are supported? `Text`? Booleans `True`, `False`? Complex atoms? Function invocations?

Reply:
An argument can be any Enso expression that can be interpreted in the context of the visualization module